Neos.ContentRepository.Core/Classes/DimensionSpace/DimensionSpacePoint.php

```diff
@@ -93,6 +93,11 @@ final public static function fromUriRepresentation(string $encoded): self
         return self::instance(json_decode(base64_decode($encoded), true));
     }
 
+    final public function toUriRepresentation(): string
+    {
+        return base64_encode(json_encode($this->coordinates, JSON_THROW_ON_ERROR));
```

Comment:
The IDE will complain that this is missing a `@throws JsonException` and I agree to it – or rather: We should never throw JsonExceptions from core domain code IMO.

For the DSP I would suggest to introduce `public function toJson(): string`, catch any JsonException in there and rethrow as RuntimeException and then use it here and everywhere else.

Same for `fromUriRepresentation`.

(could be done in a separate PR ofc)

Reply:
Knowing this is currently an already big enough of a change i didnt want to introduce anything new and seeing `fromUriRepresentation` already existed this was just something that was missed.

Lets do it in a followup: https://github.com/neos/neos-development-collection/issues/5072